app/models/bookmark.rb

```diff
@@ -46,7 +46,8 @@ def self.select_type(bookmarks_relation, type)
   validates :name, length: { maximum: 100 }
 
   def registered_bookmarkable
-    Bookmark.registered_bookmarkable_from_type(self.bookmarkable_type)
+    type = Bookmark.polymorphic_class_for(self.bookmarkable_type).name
```

Comment:
Could this line be a part of `Bookmark.registered_bookmarkable_from_type`? I think this method is called from `lib/plugin/instance.rb` too, but I am not sure if it applies/makes sense.

Reply:
Yep Im not yet settled on this, I used to do this before I namespaced chat, to separate stuff. However now I could either remove it or namespace it under discourse.

Reply:
Yep that might be better 🤔 will check :+1: